### PR TITLE
Add hackathon blog

### PR DESCRIPTION
**Pull request by ericgregory**
Adds blog summarizing hackathon projects shared in April 24 community meeting. Should go out once wash 0.28 releases.

### CODE REVIEW

File: blog/hackathon-llama-3-wit2wadm/index.mdx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+[Roman Volosatovs](https://github.com/rvolosatovs) is the driving force behind [wRPC](https://github.com/wrpc/wrpc), a framework for Wasm component-native RPC that helps remote components communicate via WIT interfaces. With v1.0, wasmCloud uses wRPC to link components at runtime over the lattice&mdash;in short, wRPC is the secret sauce that makes it possible to run distributed components seamlessly.
+
+For the Hackathon, Roman demonstrated Go working with wRPC, showing requests served by a Go program via wRPC. In the [wRPC repo](https://github.com/wrpc/wrpc), you can find examples for NATS transport over wRPC in [Go](https://github.com/wrpc/wrpc/tree/main/examples/go) as well as [Rust](https://github.com/wrpc/wrpc/tree/main/examples/rust).
+
+## wit2wadm: Generate YAML from WIT
+
+Everyone loves writing YAML manifests, so it's difficult to understand why Brooks would want to automate the process. Nevertheless, he followed his inscrutable whims to create [**wit2wadm**](https://github.com/brooksmtownsend/wit2wadm), a Rust binary and library that automatically generates Wadm manifests from provided WIT, enabling a smooth, speedy path from build to deploy with wasmCloud.
```
Comment (brooksmtownsend) [inline blog/hackathon-llama-3-wit2wadm/index.mdx:69]: 😆

**Review by brooksmtownsend — APPROVED** [review]
This looks wonderful 🚀 !

File: blog/hackathon-llama-3-wit2wadm/index.mdx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+---
+
+![wasmCloud 1.0 Hackathon](/img/hackathon-pt-1.jpg)
+
+With [v1.0 in the wild](https://wasmcloud.com/blog/wasmcloud-1.0-webassembly-apps-production-any-cloud-any-edge), wasmCloud maintainers are hacking on some big ideas, putting the new release through its paces, and generally recharging their engineering mana with some wildly creative Wasm projects. In last week's community call, maintainers and WebAssembly community contributors shared some of what they've been working on, including...
+
+* An implementation of Llama 3 using Ollama on wasmCloud
+* Workspaces on Wasm&mdash;a workspace manager for components
+* A tool to automatically generate YAML manifests for Wadm from WIT
+* Wasm-native RPC working with Go
```
Comment (rvolosatovs) [inline blog/hackathon-llama-3-wit2wadm/index.mdx]:
```suggestion
* Go support for wRPC
```

File: blog/hackathon-llama-3-wit2wadm/index.mdx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Speaking of modular Wasm tooling, Taylor also showed a new plugin system for wasmCloud Shell (wash), with plugins implemented as components. 
+
+The system works by composing an adapter component with the component for a given plugin. Once composed, a user can execute the plugin as a subcommand to `wash`. Plugins themselves can be sourced from local files, over HTTP, or from OCI registries. 
+
+What's particularly cool about the component-based approach is that very little of it is actually specific to wasmCloud or `wash`&mdash;so little, in fact, that it remains largely functional when run with Wasmtime. So in addition to a plugin system for `wash`, Taylor has created a more generalizable component-driven approach to modular CLIs.
+
+Plugins for `wash` are *already* merged into wash 0.28, so you can expect to hear more about them very, very soon. 
+
+## Wasm-native RPC&mdash;now with Go
```
Comment (rvolosatovs) [inline blog/hackathon-llama-3-wit2wadm/index.mdx]:
```suggestion
## Wasm component-native RPC&mdash;now with Go
```

**Review by CazJT — APPROVED** [review]
🤘

**Pull request merged**